source/SpinalHDL/Data types/Int.rst

```diff
@@ -150,15 +150,22 @@ Logic
 
 .. note::
 
-   ``x rotateLeft y`` and ``x rotateRight y`` are also valid syntax.
+   ``x rotateLeft y`` and ``x rotateRight y`` are also valid syntax.  Due to
+   the Scala languge able to infer a method with argument invocation from
```

Comment:
Do we know what is the name Scala ascribes to the mechanism in those specific cases ?  inferred maybe the wrong label here, does not exactly feel declarative if some situations exists where it does not occur, but with the addition of parenthesis and a full-stop character it would always occur.

Maybe `interpret the meaning of`  would replace the word `infer`.

The purpose of the statement is to inform a new user to Scala language that the cause of the difference in notation is an aspect of the Scala language not SpinalHDL and there is an equivalence of purpose.  So the reader knows at that point they can research elsewhere (outside of SpinalHDL docs) for more info about that.

Reply:
Scala calls it "method invocation" https://docs.scala-lang.org/style/method-invocation.html

Reply:
+1 to remove it then